specs/Modal.spec.js

```diff
@@ -271,6 +271,34 @@ describe('State', () => {
     expect(!isBodyWithReactModalOpenClass()).toBeTruthy();
   });
 
+  it('supports adding/removing multiple document.body classes', () => {
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A B C'
+    });
+    expect(document.body.classList.contains('A', 'B', 'C')).toBe(true);
+    unmountModal();
+    expect(document.body.classList.contains('A', 'B', 'C')).toBe(false);
+  });
+
+  it('does not remove shared classes if more than one modal is open', () => {
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A'
+    });
+    renderModal({
+      isOpen: true,
+      bodyOpenClassName: 'A B'
+    });
+
+    expect(isBodyWithReactModalOpenClass('A B')).toBe(true);
+    unmountModal();
+    expect(isBodyWithReactModalOpenClass('A B')).toBe(false);
+    expect(isBodyWithReactModalOpenClass('A')).toBe(true);
+    unmountModal();
+    expect(isBodyWithReactModalOpenClass('A')).toBe(false);
```

Comment:
Sure, I can certainly change the the tests to use the `toBeTruthy()` method, but may I ask why it is preferred?

While working with these tests, the terseness of using `toBeTruthy()` on bang methods methods—instead of something like `toBe(false)`—made it more difficult to see what I was asserting.